js/parts/PlotLineOrBand.js

```diff
@@ -54,6 +55,11 @@ Highcharts.PlotLineOrBand.prototype = {
 		// plot line
 		if (width) {
 			path = axis.getPlotLinePath(value, width);
+
+			if (typeof formatter === 'function') {
+				path = formatter(path);
```

Comment:
This should probably be `path = formatter.call(context, path)`. Just not sure what to set the context to.